### ISSUE TITLE
Add accessibility options in Ads settings for visually impared users 

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#12683]
## Description 

A blind user  on Community brought to my attention that it's very difficult for them to know when an ad has appeared. At this time, there is no way to set any auditory (or otherwise) cues to signal that an ad notification (specifically) has appeared. [In the users own words](https://community.brave.com/t/screen-reader-accessability-for-blind-users/178861):

> Build in better accessibility for the blind. Right now I have to randomly swipe into the ad and realize it is an ad, then click it on time.

Additional details/considerations discussed in internal conversation here:
https://bravesoftware.slack.com/archives/C43LW5U2H/p1605136676078500

cc @jsecretan @bradleyrichter @jenn-rhim

**Comment by tmancey** [comment]
@Brave-Matt Just waiting sign-off from the design team, and then we will merge an update for custom ad notifications. This version has better accessibility options for visually impaired users. Would this user be willing to test nightly once we merge?

**Comment by tmancey** [comment]
Closing as resolved as part of https://github.com/brave/brave-browser/issues/13132
